python/ray/ml/predictors/integrations/lightgbm/lightgbm_predictor.py

```diff
@@ -31,27 +36,29 @@ def from_checkpoint(cls, checkpoint: Checkpoint) -> "LightGBMPredictor":
                 ``LightGBMTrainer`` run.
 
         """
-        raise NotImplementedError
+        path = checkpoint.to_directory()
+        bst = lightgbm.Booster(model_file=os.path.join(path, "model.xgb"))
```

Comment:
Why `.xgb`? Btw, should these be constants (probably imported from the `LightGBMTrainer` file?)